indico/modules/events/payment/templates/emails/payment_inconsistency_email_to_manager.txt

```diff
@@ -1,7 +1,19 @@
-Dear manager of the event {{ event.title }},
+{% extends 'emails/base_i18n.txt' %}
 
-It appears that the payment of the registration with ID {{ registration.friendly_id }} has an incorrect
-amount of money (received {{ format_currency(amount, currency, locale='en_GB') }} while expecting {{ registration.render_price() }}).
-Please, check with the registrant to see if there was some misunderstanding.
+{% block subject -%}
+    {% trans %}Payment inconsistency{% endtrans %}
+{%- endblock %}
 
-Regards.
+{% block header -%}
+    {% trans title=event.title %}Dear manager of the event {{ title }},{% endtrans %}
+{%- endblock %}
+
+{% block body %}
+{%filter dedent -%}
```

Comment:
```suggestion
{% filter dedent -%}
```

(maybe uses the regex searches i sent you again :p)

Reply:
I DID.... did I forget to push at some point?? wtf...